csv-data-management/info.json

```diff
@@ -772,30 +772,105 @@
           "editable": true,
           "value": "",
           "options": [
+            "JSON Field",
             "Attachment IRI",
             "File IRI"
-          ]
-        },
-        {
-          "title": "Reference ID",
-          "type": "text",
-          "name": "value",
-          "required": true,
-          "visible": true,
-          "editable": true,
-          "value": ""
-        },
-        {
-          "title": "Provide CSV Filename",
-          "type": "text",
-          "name": "csvFileName",
-          "required": true,
-          "visible": true,
-          "editable": true,
-          "value": "",
-          "placeholder": "filename - without file extension",
-          "description": "Provide name for resulant CSV file. This file will be compressed",
-          "tooltip": "Provide name for resulant CSV file. This file will be compressed"
+          ],
+          "onchange": {
+            "JSON Field": [
+              {
+                "title": "JSON Data",
+                "type": "json",
+                "name": "json_data",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "",
+                "description": "Provide the JSON data that you want to change to CSV",
+                "tooltip": "Provide the JSON data that you want to change to CSV"
+              },
+              {
+                "title": "Provide CSV Filename",
+                "type": "text",
+                "name": "csvFileName",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "",
+                "placeholder": "filename - without file extension",
+                "description": "Provide name for resulant CSV file. This file will be compressed",
+                "tooltip": "Provide name for resulant CSV file. This file will be compressed"
+              },
+              {
+                "title": "Record Path",
+                "type": "text",
+                "name": "record_path",
+                "required": false,
+                "visible": true,
+                "editable": true,
+                "value": "",
```

Comment:
give a placeholder to understand what format of input you are expecting

Reply:
> give a placeholder to understand what format of input you are expecting

Done